### PR TITLE
add real metric list

### PR DESCRIPTION
**Pull request by spasovski**
Populates the left menu with the real charts. Uses 'name' for now but we should have something more user-readable.

### CODE REVIEW

File: distributionviewer/core/static/js/app/components/views/chart-menu.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export class ChartMenu extends React.Component {
+  render() {
+    return (
+      <nav className="filters">
+        <ul>
+          {this.props.items.map(metric => {
+            return (
+              <li key={`menu-${metric.name}`}>
+                <a href={`/metric/${metric.name}/`}>{metric.name}</a>
```
Comment (robhudson) [inline distributionviewer/core/static/js/app/components/views/chart-menu.js]: If this lands after mine, that would be `/chart/${metric.id}/`.

Comment (robhudson) [inline distributionviewer/core/static/js/app/components/views/chart-menu.js]: Does React have any concept of a reverse URL resolver so you can define your URLs in one place?

Comment (openjck) [reply distributionviewer/core/static/js/app/components/views/chart-menu.js]: It used to, but it [was removed](https://github.com/reactjs/react-router/issues/1514#issuecomment-122011215). We could consider using [react-router-reverse](https://github.com/ngokevin/react-router-reverse).

**Comment by openjck** [comment]
This is great. Only suggestion is that we also rename the class to _chart-menu_ or similar to match the components.

**Comment by spasovski** [comment]
Yea agreed on the rename. I'd like to scrap everything related to 'filtered' actually and just set a body class that has the concept of 'page-type' as in `.page-chart-list` etc.

File: distributionviewer/core/static/js/app/components/views/chart-menu.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+          })}
+        </ul>
+      </nav>
+    );
+  }
+}
+
+ChartMenu.propTypes = {
+  items: React.PropTypes.array.isRequired,
+}
```
Comment (robhudson) [inline distributionviewer/core/static/js/app/components/views/chart-menu.js]: The trailing comma! No semi-colon! It hurts my eyes!  👀 💥

**Comment by openjck** [comment]
> Yea agreed on the rename. I'd like to scrap everything related to 'filtered' actually and just set a body class that has the concept of 'page-type' as in .page-chart-list etc.

Fine by me. I misunderstood the purpose of the sidebar when calling it _filtered_.

**Comment by openjck** [comment]
Looks good. So excited to have this!

**Pull request merged**